web-registry/src/pages/CreditClassDetailsSimple.tsx

```diff
@@ -0,0 +1,313 @@
+            {count > 1 && (
+              <Link
+                sx={{ display: 'flex', alignItems: 'center' }}
+                href="TODO"
+                target="_blank"
+              >
+                <Label sx={{ mr: 2 }}>{`+ ${count - 1} more`}</Label>{' '}
+                <SmallArrowIcon className={styles.arrow} />
+              </Link>
+            )}
+          </Box>
+        }
+      />
+    ) : (
+      <></>
+    );
+  };
+
+  interface LineItemProps {
+    label: string;
+    data: string | JSX.Element;
+  }
+
+  const LineItem = ({ label, data }: LineItemProps): JSX.Element => {
+    return !!data ? (
+      <Box
+        sx={{
+          width: { xs: '100%', sm: '50%' },
+          pr: 4,
+        }}
+        className={styles.marginBottom}
+      >
+        <Label className={styles.label}>{label}</Label>
+        <Description className={styles.description}>{data}</Description>
+      </Box>
+    ) : (
+      <></>
+    );
+  };
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        bgcolor: 'primary.main',
+      }}
+    >
+      <div className="topo-background-alternate">
+        <Section classes={{ root: styles.sectionPadding }}>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: { xs: 'column', md: 'row' },
+              justifyContent: 'space-between',
+            }}
+          >
+            <Box
+              sx={{
+                mr: 12,
+              }}
+            >
+              <Box className={styles.marginBottom}>
+                <Label
+                  sx={{
+                    fontSize: { xs: 12, sm: 14 },
+                    color: 'info.dark',
+                    mb: 2,
+                  }}
+                >
+                  credit class
+                </Label>
+                <Title variant="h3">{ledgerClass.class_id}</Title>
+              </Box>
+              {metadata?.['schema:description'] && (
+                <ReadMore>{metadata?.['schema:description']}</ReadMore>
+              )}
+              <Box sx={{ display: 'flex', flexWrap: 'wrap' }}>
+                <LineItem
+                  label="credit type"
+                  data={toTitleCase(ledgerClass.credit_type.name)}
+                />
+                {metadata?.['regen:sourceRegistry']?.['schema:name'] && (
+                  <LineItem
+                    label="registry"
+                    data={
+                      <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                        <Description
+                          sx={{ mr: 1 }}
+                          className={styles.description}
+                        >
+                          {metadata?.['regen:sourceRegistry']?.['schema:name']}
+                        </Description>
+                        <Link
+                          href={
+                            metadata?.['regen:sourceRegistry']?.['schema:URL']
```

Comment:
```suggestion
                            metadata?.['regen:sourceRegistry']?.['schema:url']
```

[schema:URL](https://schema.org/URL) is a datatype, we need the property here: [schema:url](https://schema.org/url)

Reply:
thanks. done.